param_passing/proposal.md

```diff
@@ -0,0 +1,388 @@
+# Proposal for parameter passing
+
+* **Authors**:
+Jenn Nguyen `<jenn@openrobotics.org>`,
+Addisu Taddese `<addisu@openrobotics.org>`
+* **Status**: Draft
+* **SDFormat Version**: 1.7
+* **`libsdformat` Version**: 10
+
+## Introduction
+
+**Purpose statement**
+
+This proposal suggests extending the [composition behavior in SDFormat 1.7](
+  http://sdformat.org/tutorials?tut=composition#including-a-model)
+(i.e., using the `<include>` tag) to pass additional arguments to SDFormat files
+which will allow a user to send custom data into a model file and reduce model
+the need for file duplication.
```

Comment:
one more rephrase: "reduce model the need for" -> "reduce the need for model"

Reply:
Oops, thank you. 159ecf5629de940006ee2945ffb15479c43a78f3